airbyte-cdk/python/type_check_and_test.sh

```diff
@@ -6,7 +6,7 @@
 # Static Type Checking
 echo "Running MyPy to static check and test files."
 # Exclude Singer for the time being.
```

Comment:
nit: can you remove that deprecated comment?

Reply:
I'll remove it later this evening or tomorrow